extension/httpforwarder/README.md

```diff
@@ -1,10 +1,15 @@
 # HTTP Forwarder Extension
-
+<!-- status autogenerated section -->
 | Status                   |           |
 | ------------------------ |-----------|
-| Stability                | [beta]    |
+| Stability                | [beta]   |
+| Supported pipeline types |    |
```

Comment:
I think we should make `mdatagen` skip this if `type` is `extension`. Wdyt?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/20236 is open.

Reply:
Merged that one :) can you rebase and regenerate this?

Reply:
Merged that one :) can you rebase and regenerate this?